bc_gym_planning_env/envs/base/action.py

```diff
@@ -25,3 +25,29 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return not self.__eq__(other)
+
+    def processed_command(self):
```

Comment:
I don't know what it is, but this function looks shady.. It returns different hardcoded numbers depending on the shape and is in 'base' folder (assuming it supposed to be quite generic)

Reply:
agreed. Making this class can handle both discrete and continuous action seems non-trivial at this point. Maybe I should revert this change and create a separate PR to focus on solving this issue